en/components/general/framework-and-features/data-analysis.md

```diff
@@ -22,12 +39,10 @@ We currently support the following Chart types:
 - ScatterPoint, ScatterBubble, ScatterLine. In order to show meaningful Scatter Bubble Chart we disable the preview when the data is not in valid format.
 - [Pie Chart](../../pie-chart.md)
 
-> Note: This functionality will be introduced as a Directive in order to ease the configuration and limit the required code at minimum
-
 ### Conditional Cell Formatting 
 If you have a Grid with thousands of rows of data it would be very difficult to see patterns and trends just from examining the raw information. Similar to charts and sparklines, `Conditional formatting` provides another way to visualize data and make it easier to understand. 
 
-Understanding conditional formatting - it allows you to automatically apply formatting such as colors and data bars to cells based on `their value` in the range selection. The [sample below](#demo) demonstrates how you can configure the Grid to apply `Conditional Formatting`. It depends on the `Conditional formatting selection type` what condition `rules` will be shown. Below you will find the predefined styles (presets) that you can use in order to quickly apply conditional formatting to your data.
+Understanding conditional formatting - it allows you to automatically apply formatting such as colors and data bars to cells based on `their value` in the range selection. The [sample below](#demo) demonstrates how you can configure the Grid to apply `Conditional Formatting`. It depends on the `Conditional formatting selection type` what condition `rules` will be shown.  Below you will find the predefined styles (presets) that you can use in order to quickly apply conditional formatting to your data. The formatting of a range clears when performing formatting on different range or through the clear button. The clear button activates when there is an applied formatting and removes the formatting from the whole range.  
```

Comment:
it allows for applying formatting such as

Reply:
The formatting of a range gets cleared when performing formatting on different range or through the clear button.

Reply:
The clear button is only active when there is an applied formatting.